### PR TITLE
Updated mock seed data to reflect changes agreed upon by DS, FE, and BE

### PR DESCRIPTION
**Pull request by MattBokovitz1**
Updated mock seed data fields to match Data Science's schema.

### CODE REVIEW

File: data/seeds/003-cases.js [hunk 1]
```diff
@@ -8,305 +8,335 @@ exports.seed = function (knex) {
...
+          type_of_violence_experienced: 'Not Applicable',
+          application_type: 'initial',
+          protected_ground: 'true',
         },
         {
-          id: 'A095-094-694 ',
+          case_id: 'A095-094-694 ',
           case_url:
             'pdf/139810918-Rony-Rene-Anariba-A095-094-694-BIA-Nov-16-2012.pdf',
           judge_name: 'Charles K. Adkins-Blanch',
```
Comment (odst0016) [inline data/seeds/003-cases.js:257]: I cant find the change to judge in this one.

**Review by odst0016 — CHANGES_REQUESTED** [review]
Looks like one of the changes wasn't made.

File: data/seeds/003-cases.js [hunk 2]
```diff
@@ -8,305 +8,335 @@ exports.seed = function (knex) {
...
-          one_year_guideline: 'false',
-          case_status: 'Closed',
-          judge_decision: 'Remanded',
+          case_filed_within_one_year: false,
+          // case_status: 'Closed',
+          case_outcome: 'Remanded',
+          applicant_sex: 'Female',
+          type_of_violence_experienced: 'Not Applicable',
+          application_type: 'initial',
+          protected_ground: true,
```
Comment (kristapants) [inline data/seeds/003-cases.js:184]: This true should be a string, I think.

**Review by kristapants — COMMENTED** [review]
One comment left inline for a boolean that may need to be a string.

File: data/seeds/003-cases.js (hunk 2 above)
Comment (MattBokovitz1) [reply data/seeds/003-cases.js]: Thanks for pointing this out, I fixed this.

File: data/seeds/003-cases.js (hunk 1 above)
Comment (MattBokovitz1) [reply data/seeds/003-cases.js]: Thanks for point this out, I changed this and made a new commit.

**Review by odst0016 — APPROVED** [review]

**Review by kristapants — APPROVED** [review]
These changes match with the changes I've made to the table builder. This should bring us into line across BE and DS

**Pull request merged**